frontendservice/src/views/specificShop.vue

```diff
@@ -142,6 +143,25 @@ export default {
             });
           }
         });
+
+        self.getWaitingTime();
+
+        if(this.flag) {
```

Comment:
karishma please change the flag names in your pr as per the update-flag-names pr i just sent out.

Reply:
I pulled the code from the master and implemented the waiting time. Can you merge your PR after my PR gets merged please if there's anything like that?

Reply:
karishma pull from master and change the flag name here as per it.

Reply:
Done. Please check once.

Reply:
yes all is good :)